### DISCUSSION TITLE
Copy Multiple Links or Magnets

### DISCUSSION THREAD

**Discussion by aerogurl** [LivingWithHippos/unchained-android#367] [Ideas]
If I'm wanting to upload a whole season of TV show links into Unchained, I can click each link or copy/paste each individual link but both are  time-consuming.  I can copy/paste all links at once into Real Debrid so there is a workaround but I'd like to see Unchained a little more like RD where you can upload multiple links.. Just a suggestion.
![20240225_005132](https://github.com/LivingWithHippos/unchained-android/assets/124489463/a2e5815e-4b6e-47f9-a8c6-2586bafe4ff4)

**Comment by LivingWithHippos** [comment]
have you tried copying and pasting into Unchained? I think I added the support for multiple links

**Reply by Invictaz** [reply]
Nope it was not added. See my issue #352 it doesn't work.

**Comment by LivingWithHippos** [comment]
Ok I rechecked and it does not work, I must have broken it because I remember it worked. Shouldn't be hard to fix since it's already implemented

**Reply by Invictaz** [reply]
Thanks for checking
